modules/imgproc/test/test_imgwarp_strict.cpp

```diff
@@ -150,7 +150,7 @@ void CV_ImageWarpBaseTest::generate_test_data()
     while (depth == CV_8S || depth == CV_32S)
         depth = rng.uniform(0, CV_64F);
 
-    int cn = rng.uniform(1, 4);
+    int cn = rng.uniform(1, 5);
```

Comment:
The change triggers massve OpenCL code compilation errors for the new case. Some check should be added on OpenCL side. See attached log.
[Uploading test.log…]()